tests/testthat/test-anovarepeatedmeasuresbayesian.R

```diff
@@ -313,15 +313,16 @@ test_that("Model Averaged Posterior Summary table results with interactions matc
 			 "contcor1", "f &amp; contcor1", -0.197666349533573, -0.0103632975300062,
 			 0.0926887994083915, 0.173741376044989, "facGender<unicode><unicode><unicode>contcor1",
 			 "m &amp; contcor1", -0.175839724777071, 0.0103632975300062,
-			 0.0926887994083915, 0.195568000801491, "", "f &amp; Level 1",
+			 0.0926887994083915, 0.195568000801491, "", "Level 1 &amp; f",
 			 -0.175438316624131, -0.0141219719774659, 0.0801968484745378,
-			 0.14453349565803, "facGender<unicode><unicode><unicode>RM_Factor_1",
-			 "f &amp; Level 2", -0.146415682789101, 0.0141219719774659, 0.0801968484745378,
-			 0.173556129493059, "", "m &amp; Level 1", -0.147458708170939,
-			 0.0141219719774659, 0.0801968484745378, 0.174923594381005, "",
-			 "m &amp; Level 2", -0.176602668873463, -0.0141219719774659,
-			 0.0801968484745378, 0.145779633678481, "", "Level 1", 0.0194629065393367,
-			 0.185110395127867, 0.0823440335153794, 0.349609596761152, "contcor1<unicode><unicode><unicode>RM_Factor_1",
-			 "Level 2", -0.351014476294011, -0.185110395127867, 0.0823440335153794,
-			 -0.0208677860721954, ""))
+			 0.14453349565803, "RM_Factor_1<unicode><unicode><unicode>facGender",
+			 "Level 1 &amp; m", -0.146415682789101, 0.0141219719774659, 0.0801968484745378,
+			 0.173556129493059, "", "Level 2 &amp; f", -0.145880989401819,
+			 0.0141219719774659, 0.0801968484745378, 0.174276368695733, "",
+			 "Level 2 &amp; m", -0.176240524266883, -0.0141219719774659,
+			 0.0801968484745378, 0.143916833830669, "", "Level 1 &amp; contcor1",
+			 0.0208677860721955, 0.185156290606396, 0.0824454986811013, 0.349609596761152,
+			 "RM_Factor_1<unicode><unicode><unicode>contcor1", "Level 2 &amp; contcor1",
+			 -0.351014476294011, -0.185156290606396, 0.0824454986811013,
+			 -0.0222726656050543, ""))
```

Comment:
Note: because the order of the formula changed, the order of the levels also changed. The values here should be identical to what they were before (up to a tolerance).